pkg/compose/build.go

```diff
@@ -129,6 +129,9 @@ func (s *composeService) build(ctx context.Context, project *types.Project, opti
 		if options.Quiet {
 			options.Progress = progress.ModeQuiet
 		}
+		if options.Progress == "" {
```

Comment:
I believe this doesn't work since the default value is `string(buildkit.AutoMode)`.

My docker compose is v2.33.1 and this envar doesn't work.